File: src/test/fixtures/csharp/MagicString.cs
class CleanStrings
{
// clean — not flagged by magic string
    static string CleanValues(string name) => "user " + name;
}

class FlaggedStrings
{
// flagged — magic string
    static int FlaggedMagicString(string status)
    {
        if (status == "pending") return 1;
        if (status == "pending") return 2;
        return 0;
    }
}

File: src/test/fixtures/fsharp/MagicString.fs
module MagicString

// clean — not flagged by magic string
let cleanValues (name: string) = sprintf "user %s not found" name

// flagged — magic string
let flaggedMagicString (status: string) =
    if status = "pending" then 1
    elif status = "pending" then 2
    else 0
